javaslang/src/main/java/javaslang/concurrent/Future.java

```diff
@@ -539,7 +539,7 @@
     /**
      * Blocks the current Thread until this Future completed or returns immediately if this Future is already completed.
      */
```

Comment:
Please add a `@return this Future instance` to the Javadoc and test it with `mvn javadoc:javadoc`. We need to eliminate all doclint warnings, otherwise a release fails next time.